src/app.json.template

```diff
@@ -10,12 +10,8 @@
         "odt"
     ],
     "companyName": "[ COMPANY NAME HERE ]",
-    "defaultLocale": "en-US",
-    "supportedLocales": [
-        "en-US",
-        "fr-FR"
-    ],
-    "minUploadSize": 30720,
+    "defaultLocale": "en-EU",
+    "minUploadSize": 3720,
```

Comment:
@jgodi check with Mitesh... there is a minimum document size that the REX parser allows, and it bombs if the size is less. It is nominal, but might be greater that 3KB.